src/Core/WebServer.cs

```diff
@@ -13,6 +13,7 @@
 using SwarmUI.WebAPI;
 using System.IO;
 using System.Net;
+using System.Text.RegularExpressions;
```

Comment:
(nit) accidental import there

Reply:
dangit
i had a regex here at one point but i got rid of it and forgot this, will fix